.github/workflows/integrationTest.yaml

```diff
@@ -0,0 +1,51 @@
+# Test the functionality of the rgbpp-sdk packages.
+
+name: integrationTest
```

Comment:
Would you update the name of the workflow to align it with other workflows?
Some suggestions: "Test RGBPP Lifecycle", "Test Lifecycle", "Integration Testing", "Integration Tests".

Reply:
> Integration Tests
+1